libcontainer/integration/exec_test.go

```diff
@@ -1446,18 +1446,18 @@ func TestRootfsPropagationSlaveMount(t *testing.T) {
 	}
 
 	err = container.Run(pconfig)
-	stdinR.Close()
-	defer stdinW.Close()
+	_ = stdinR.Close()
+	defer func() { _ = stdinW.Close() }()
 	ok(t, err)
 
 	// Create mnt1host/mnt2host and bind mount itself on top of it. This
 	// should be visible in container.
 	dir2host, err := ioutil.TempDir(dir1host, "mnt2host")
 	ok(t, err)
-	defer os.RemoveAll(dir2host)
+	defer remove(dir2host)
 
 	err = unix.Mount(dir2host, dir2host, "bind", unix.MS_BIND, "")
-	defer unmountOp(dir2host)
+	defer func() { _ = unmountOp(dir2host) }()
```

Comment:
same here -- since no caller of unmountOp() cares about its return, it's better to change it to not return anything.

Reply:
updated 👍